src/exercises/WorkingWithAnAPI.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react";
+import MemeCard from "../components/MemeCard";
+import { fetchData } from "../utils/fetchData";
+
+export default function WorkingWithAnAPI() {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    fetchData().then((e) => setData(e));
+  }, []);
+  return (
+    <div className="w-full h-screen mt-32">
+      <div className="w-full justify-center flex">
+        <button
+          className="bg-red-500 rounded-full shadow-lg p-4 w-fit"
+          onClick={() => fetchData().then((e) => setData(e))}
```

Comment:
Here it would be better if we have `data` to represent the response data instead of having `e` only.